src/Task1/Vote.js

```diff
@@ -20,8 +20,19 @@ class Vote extends Component {
       }
```

Comment:
@Poorvi-3pillar This method created 
```setState(state) {
        window.localStorage.setItem('state', JSON.stringify(state));
        super.setState(state);
      }
```
is not being called anywhere in the code. I don't we need this. Make changes accordingly